net/ftpproxy/src/opnsense/mvc/app/controllers/OPNsense/FtpProxy/Api/SettingsController.php

```diff
@@ -0,0 +1,305 @@
+<?php
```

Comment:
Was this developed against 16.7.x or master branch? There are some API twists in the master branch we plan to bring to 16.7.6. I don't know if it affects this. The correct one for this plugin would be 16.7.x code at the moment.

CC @AdSchellevis